inventory/environ.py

```diff
@@ -217,58 +294,37 @@ def overrideEnvironmentVars(vars_scope):
     vars_scope["splunk"]["s2s"]["port"] = int(os.environ.get('SPLUNK_S2S_PORT', vars_scope["splunk"]["s2s"]["port"]))
     vars_scope["splunk"]["secret"] = os.environ.get('SPLUNK_SECRET', vars_scope["splunk"]["secret"])
     vars_scope["splunk"]["hec_token"] = os.environ.get('SPLUNK_HEC_TOKEN', vars_scope["splunk"]["hec_token"])
-    if "shc" not in vars_scope["splunk"]:
```

Comment:
This section was removed from `overrideEnvironmentVars()`, split into its own function `getSearchHeadClustering()`, and moved directly into `getDefaultVars()`